x/auth/legacy/v040/migrate.go

```diff
@@ -11,9 +12,15 @@ import (
 // convertBaseAccount converts a 0.39 BaseAccount to a 0.40 BaseAccount.
 func convertBaseAccount(old *v039auth.BaseAccount) *v040auth.BaseAccount {
 	var any *codectypes.Any
-	// If the old genesis had a pubkey, we pack it inside an Any. Or else, we
-	// just leave it nil.
-	if old.PubKey != nil {
+
+	_, ok := old.PubKey.(*multisigtypes.LegacyAminoPubKey)
+
+	// If pubkey is multisig type, then leave it as nil for now
```

Comment:
Could you reference the issue for doing this?

Reply:
Line 19